bioconda_utils/build.py

```diff
@@ -259,6 +260,27 @@ def get_subdags(dag, n_workers, worker_offset):
     return subdags
 
 
+def check_native_platform_skippable(recipe_folder: str, recipe: str):
+    """
+    Given a recipe, check this recipe should skip in current platform or not.
+
+    Arguments:
+      recipe_folder: Directory containing possibly many, and possibly nested, recipes.
+      recipe: recipe name
+
+    Returns:
+      Return True if current native platform are not included in recipe's additional platforms (no need to build).
+    """
+    recipe_obj = _recipe.Recipe.from_file(recipe_folder, recipe)
+    native_platform = utils.RepoData().native_platform()
+    # On linux-aarch64 env, only build recipe with linux-aarch64 extra_additional_platforms
+    if native_platform == "linux-aarch64":
+        if "linux-aarch64" not in recipe_obj.extra_additional_platforms:
+            logger.info("BUILD SKIP: skipping %s for %s platform", recipe, native_platform)
```

Comment:
I would move the log message into the if statement where this function is used below.

Reply:
Yes, I also considered before. So the `native_platform` should be moved out of function to help log print, the testcase can be simpler